src/common/components/FieldSelection/FieldSelection.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+import { Field } from 'redux-form';
+
+import { Selection } from '@folio/stripes/components';
+
+const TETHER_CONFIG = {
+  attachment: 'middle center',
+};
+
+const EMPTY_OPTION = { label: '', value: null };
+
+const FieldSelection = ({ dataOptions, labelId, ...rest }) => (
+  <Field
+    dataOptions={rest.required ? dataOptions : [EMPTY_OPTION, ...dataOptions]}
+    disabled={rest.readOnly}
+    fullWidth
+    label={<FormattedMessage id={labelId} />}
```

Comment:
what if `labelId` is missed?

Reply:
the exception will be raised. Tests will be failing.

Reply:
However, if `label` is passed instead of `labelId` all works fine. So I prefer to keep it as is